LICENSE

```diff
@@ -652,7 +652,7 @@ Also add information on how to contact you by electronic and paper mail.
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    <program>  Copyright (C) <year>  <name of author>
+    TabNews  Copyright (C) 2020  Filipe Deschamps
```

Comment:
Será que não é melhor deixar o nome do programa como `tabnews.com.br`, assim como é o nome do repositório? Isso porque aqui temos exatamente o código da aplicação web do TabNews, que se diferencia de outras bibliotecas que lançamos/lançaremos, e que são/serão mais interessantes para quem quiser criar projetos com partes similares ao TabNews.

Sobre o ano, acho que aqui entra a faixa, não é? Daí ficaria `2020-2024`, ou `2021-2024`, se for contar quando essa licença foi adotada, o que me parece mais correto.

Por fim, como autor, aqui sim talvez se encaixe o nome TabNews como sendo a instituição autora do código.

Estou deixando a sugestão abaixo, mas não sei se deveria ficar assim. Tenho dúvidas mesmo nos três pontos.

```suggestion
    tabnews.com.br  Copyright (C) 2021-2024  TabNews
```

Reply:
A escolha de usar o `TabNews` como autor ao invés do @filipedeschamps, prefiro deixar para vocês (ambos fazem sentido, isso fica mais a critério dos mantenedores ao meu ver) 🙋🏻‍♂️

Sobre a questão do `2021-2024`, vou deixar aqui uma ideia que uso nos meus projetos a fim de evitar todo ano ter que voltar na licença manualmente: `2020-current` (ou também `2024-present`).